### PR TITLE
Dependencies

### PR DESCRIPTION
**Pull request by jmthomas**

### CODE REVIEW

File: openc3-minio/Dockerfile [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ARG OPENC3_DEPENDENCY_REGISTRY=docker.io
-FROM ${OPENC3_DEPENDENCY_REGISTRY}/minio/minio:RELEASE.2023-10-07T15-07-38Z
+FROM ${OPENC3_DEPENDENCY_REGISTRY}/minio/minio:RELEASE.2024-01-05T22-17-24Z
```
Comment (jmthomas) [inline openc3-minio/Dockerfile:2]: Verified in IronBank

File: openc3-traefik/Dockerfile [hunk 2]
```diff
@@ -1,5 +1,5 @@
 ARG OPENC3_DEPENDENCY_REGISTRY=docker.io
-FROM ${OPENC3_DEPENDENCY_REGISTRY}/traefik:v2.10.6
+FROM ${OPENC3_DEPENDENCY_REGISTRY}/traefik:v2.10.7
```
Comment (jmthomas) [inline openc3-traefik/Dockerfile:2]: Verified in IronBank

File: openc3-minio/Dockerfile [hunk 3]
```diff
@@ -11,13 +11,6 @@ ENV CURL_CA_BUNDLE=/devel/cacert.pem
 ENV REQUESTS_CA_BUNDLE=/devel/cacert.pem
 ENV NODE_EXTRA_CA_CERTS=/devel/cacert.pem
 
-# Update packages to eliminate CVEs if we're on docker.io (not ironbank)
-RUN if [[ $OPENC3_DEPENDENCY_REGISTRY == 'docker.io' ]]; then \
-  microdnf update --nodocs -y && microdnf clean all; \
-  groupadd -g 1001 minio; \
-  useradd -r -u 1001 -m -g minio minio; \
-  fi
-
```
Comment (jmthomas) [inline openc3-minio/Dockerfile:20]: The new minio containers are now based on ubi-micro (https://github.com/minio/minio/issues/18473) which do not include a package manager. They also do not include any additional tools. So there's no packages to update and I think we can just run as USER 1001 wtihout adding the user / group (see https://github.com/minio/minio/issues/18380).

File: openc3/openc3.gemspec [hunk 4]
```diff
@@ -77,64 +77,65 @@ spec = Gem::Specification.new do |s|
...
-  s.add_development_dependency 'simplecov-cobertura', '~> 2.1'
   s.add_development_dependency 'benchmark-ips', '~> 2.9'
-  s.add_development_dependency 'ruby-prof', '~> 1.4' if RUBY_ENGINE == 'ruby' # MRI Only
   s.add_development_dependency 'rspec_junit_formatter', '~> 0.4'
+  s.add_development_dependency 'ruby-prof', '~> 1.4' if RUBY_ENGINE == 'ruby' # MRI Only
+  s.add_development_dependency 'simplecov', '~> 0.21'
+  s.add_development_dependency 'simplecov-cobertura', '~> 2.1'
 
   s.post_install_message = "Thanks for installing OpenC3!\n"
+  s.metadata['rubygems_mfa_required'] = 'true'
```
Comment (jmthomas) [inline openc3/openc3.gemspec:140]: I didn't really change anything here but they got sorted alphabetically.

**Review by ryanmelt — APPROVED** [review]
Did Single-SPA 6 not work?

**Pull request merged**